lib/gtl_formatter.rb

```diff
@@ -96,15 +109,33 @@ def self.format_cols(view, row, controller, options)
                 :icon  => icon,
                 :icon2 => icon2,
                 :text  => text}.compact
-      elsif COLUMN_WITH_ICON.keys.include?(col)
+      elsif COLUMN_WITH_BACKGROUND_ICON.key?(col)
+        # Generate html for the list icon with background
+        icon = send(COLUMN_WITH_BACKGROUND_ICON[col], record)
+        text = format_col_for_display(view, row, col)
+        item = {:title      => text,
+                :icon       => icon,
+                :background => textual_power_state_whitelisted(record[col]),
+                :text       => text}.compact
+      elsif COLUMN_WITH_ICON.key?(col)
         # Generate html for the list icon
         icon = send(COLUMN_WITH_ICON[col], record)
         text = format_col_for_display(view, row, col)
         item = {:title => text,
                 :icon  => icon,
-                :text => text}.compact
+                :text  => text}.compact
       elsif COLUMN_WITH_TIME.include?(col)
         celltext = format_time_for_display(row, col)
+      elsif COLUMN_WITH_OS_ICON.key?(col)
```

Comment:
For consistency

```suggestion
      elsif COLUMN_WITH_OS_ICON.include?(col)
```

`.key?` and `.include?` are aliases, but since everything else is using include? it looks more visually consistent this way.

Reply:
Actually, upon further review, it's about a 50/50 split.  Since `include?` can also be used for Arrays, it's a little better in terms of consistency, so I think it's better to use that term in the various branches here.